### PR TITLE
save `buco` and `bvco` on half mesh

### PR DESCRIPTION
**Pull request by ddudt**
In `VMECIO.save` we were saving the outputs `buco` and `bvco` on the full mesh, but they should be on the half mesh.

### CODE REVIEW

File: desc/vmec.py [hunk 1]
```diff
@@ -609,31 +606,31 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
         DShear = file.createVariable("DShear", np.float64, ("radius",))
         DShear.long_name = "Mercier stability criterion magnetic shear term"
```
Comment (dpanici) [inline desc/vmec.py:607]: specify mesh

File: desc/vmec.py [hunk 2]
```diff
@@ -609,31 +606,31 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
...
         DShear.units = "1/Wb^2"
-        DShear[:] = grid.compress(data["D_shear"])
+        DShear[:] = grid_full.compress(data_full["D_shear"])
         DShear[0] = 0
 
         DCurr = file.createVariable("DCurr", np.float64, ("radius",))
         DCurr.long_name = "Mercier stability criterion toroidal current term"
```
Comment (dpanici) [inline desc/vmec.py:613]: specify mesh

File: desc/vmec.py [hunk 3]
```diff
@@ -609,31 +606,31 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
...
         DCurr.units = "1/Wb^2"
-        DCurr[:] = grid.compress(data["D_current"])
+        DCurr[:] = grid_full.compress(data_full["D_current"])
         DCurr[0] = 0
 
         DWell = file.createVariable("DWell", np.float64, ("radius",))
         DWell.long_name = "Mercier stability criterion magnetic well term"
```
Comment (dpanici) [inline desc/vmec.py:619]: specify mesh

File: desc/vmec.py [hunk 4]
```diff
@@ -609,31 +606,31 @@ def save(cls, eq, path, surfs=128, verbose=1):  # noqa: C901 - FIXME - simplify
...
         DWell.units = "1/Wb^2"
-        DWell[:] = grid.compress(data["D_well"])
+        DWell[:] = grid_full.compress(data_full["D_well"])
         DWell[0] = 0
 
         DGeod = file.createVariable("DGeod", np.float64, ("radius",))
         DGeod.long_name = "Mercier stability criterion geodesic curvature term"
```
Comment (dpanici) [inline desc/vmec.py:625]: specify mesh

**Review by dpanici — CHANGES_REQUESTED** [review]
Looks good, just add to the descriptions of the ones you changed which mesh they are on to be consistent with the other descriptions changed

**Comment by ddudt** [comment]
> Looks good, just add to the descriptions of the ones you changed which mesh they are on to be consistent with the other descriptions changed

I didn't change the Mercier quantities, but I have now updated their descriptions to say they are saved on the full mesh.

**Review by dpanici — APPROVED** [review]

**Review by f0uriest — APPROVED** [review]

**Pull request merged**